packages/actor-rdf-resolve-quad-pattern-federated/lib/BlankNodeSkolemizable.ts

```diff
@@ -0,0 +1,22 @@
+import * as RDF from "rdf-js";
+
+export class BlankNodeSkolemizable implements RDF.BlankNode {
```

Comment:
Perhaps a slight misnomer, given that all blank nodes can be skolemized.

How about something like `ScopedBlankNode` or `RestrictedBlankNode` or `ReusableBlankNode` or `TraceableBlankNode`?